app/serializers/searchers/responses_searcher_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Searchers
+  # Serializes a ResponsesSearcher for search filters.
+  class ResponsesSearcherSerializer < SearcherSerializer
+    attributes :all_forms, :selected_form_ids, :selected_qings, :is_reviewed,
+      :selected_users, :selected_groups
+
+    def selected_form_ids
+      object.form_ids || []
+    end
+
+    def selected_qings
+      object.qings || []
+    end
+
+    def selected_users
+      object.submitters || []
+    end
+
+    def selected_groups
+      object.groups || []
+    end
+
+    def advanced_search_text
+      (Settings.filters_beta.present? ? object.advanced_text : object.query) || ""
+    end
+  end
+end
```

Comment:
I think you should not be using a `Searchers` namespace as it's inconsistent with other object types. e.g. there are no Models or Controllers or Decorators namespaces.

Namespaces should be about the domain, not the object type. So Odk, Results, Forms, etc. You could put this one in Results if you want to namespace it.